include/common/utils/raw_send_nbarrier.hpp

```diff
@@ -106,23 +111,15 @@ struct xetla_nbarrier_t<num_producers, num_consumers, gpu_arch::Dg2> {
 
     /// @brief Generic work-group split barrier.
     ///
-    __XETLA_API void arrive() {
-        __ESIMD_ENS::split_barrier<__ESIMD_ENS::split_barrier_action::signal>();
-    }
+    __XETLA_API void arrive() { __ESIMD_NS::barrier(); }
```

Comment:
Is there anything wrong with split_barrier_action?

Reply:
The changes here was to avoid another bug fixed in f1f010d. Changes in `raw_send_nbarrier.hpp` are reverted.